Cargo.toml

```diff
@@ -120,7 +120,7 @@ etcd-client = { git = "https://github.com/MichaelScofield/etcd-client.git", rev
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/killme2008/greptime-proto.git", rev = "a15a54a714fe117d7e9f7635e149c4eecac773fa" }
+greptime-proto = { git = "https://github.com/killme2008/greptime-proto.git", rev = "6e0f5e684875207ba460e08a5d21ab7f775bba9e" }
```

Comment:
Should be https://github.com/GreptimeTeam/greptime-proto?

Reply:
Good catch!